### ISSUE TITLE
Enforce Intercom identity verification

### ISSUE THREAD

**Issue by jgwerner** [3Blades/3blades#160]
We are identifying users on 3blades.io but need to do the same on 3blades.io:

https://app.intercom.io/a/apps/kx01msas/settings/identity-verification/web

This is important for our on-boarding campaign.

**Comment by johngriebel** [comment]
@jgwerner I think you have a typo; 3blades.io is listed twice.

**Comment by jgwerner** [comment]
@johngriebel oops! Corrected.

**Comment by raygesualdo** [comment]
This will require a change to the server payload for the `User` model to include an Intercom-specific hash of the currently logged in user's ID. This can't be enabled client-side only.

**Comment by jgwerner** [comment]
@raygesualdo yeah I saw that as well in the docs. Would have an affect on swagger doc too.
